packages/react/src/components/Notification/Notification.js

```diff
@@ -471,6 +480,17 @@ export class InlineNotification extends Component {
             {this.props.children}
           </NotificationTextDetails>
         </div>
+        {actions.primary && actions.primary.onClick && actions.primary.label && (
+          <Button
```

Comment:
Can we define something like `<NotificationActionButton>` component (probably defined as a named export in this file) so this block can just be `{actions}`?